internal/nix/eval.go

```diff
@@ -46,6 +46,21 @@ func PackageKnownVulnerabilities(path string) []string {
 	return vulnerabilities
 }
 
+func PackagePlatforms(path string) []string {
+	cmd := command("eval", path+".meta.platforms", "--json")
```

Comment:
What if instead of searching and eval'ing the platforms we just eval the derivation's outPath and check the output? For example:

```
# Bad
$ nix eval --raw nixpkgs#fuse3
error: Package ‘fuse-3.11.0’ in /nix/store/b63zicy7mkwlp4vmca056kawsrmxfr36-source/pkgs/os-specific/linux/fuse/common.nix:92 is not available on the requested hostPlatform:
         hostPlatform.config = "aarch64-apple-darwin"
         package.meta.platforms = [
...
```

vs.

```
# Good
$ nix eval --raw nixpkgs#fuse 
/nix/store/8pqmbsp8zkq8656k5yaravbaxixaaa3c-macfuse-stubs-4.4.1
```